http/action/src/main/java/io/knotx/databridge/http/action/HttpAction.java

```diff
@@ -79,45 +83,60 @@
   private final String actionAlias;
   private final HttpActionOptions httpActionOptions;
   private final ResponsePredicatesProvider predicatesProvider;
+  private final ActionLogLevel logLevel;
   private static final ResponsePredicate IS_JSON_RESPONSE = ResponsePredicate
       .create(ResponsePredicate.JSON, result -> {
         throw new ReplyException(ReplyFailure.RECIPIENT_FAILURE, result.message());
       });
 
-  HttpAction(Vertx vertx, HttpActionOptions httpActionOptions, String actionAlias) {
+  HttpAction(Vertx vertx, HttpActionOptions httpActionOptions, String actionAlias,
+      ActionLogLevel actionLogLevel) {
```

Comment:
Maybe simply `logLevel` instead of `actionLogLevel`.

Reply:
fixed